lib/ansible/playbook/taggable.py

```diff
@@ -34,11 +44,7 @@ def _load_tags(self, attr, ds):
         if isinstance(ds, list):
             return ds
         elif isinstance(ds, string_types):
-            value = ds.split(',')
```

Comment:
Not a part of the fix but figured I might as well delete some code while I was here. `str.split` always returns a `list` so the if-else below isn't needed.